custom_types.yaml

```diff
@@ -361,11 +359,9 @@ node_types:
         create:
           implementation: https://raw.githubusercontent.com/indigo-dc/tosca-types/master/artifacts/galaxy/galaxy_tools_configure.yml
           inputs:
-            galaxy_install_path: { get_property: [ HOST, install_path ] }
+            galaxy_flavor: { get_property: [ HOST, flavor ] }
```

Comment:
Hi @mtangaro,
Tests have passed OK. But I thinks that this line is not correct.
It should be:
galaxy_flavor: { get_property: [ **SELF**, flavor ] }
As you are refering the a property of this node, not a node of the GalaxyPortal node where it is hosted.